src/tests/ftest/cart/corpc/cart_corpc_five_node.py

```diff
@@ -37,7 +39,7 @@ class CartCoRpcFiveNodeTest(Test):
     """
     Runs basic CaRT CoRPC tests
 
-    :avocado: tags=all,corpc,five_node
+    :avocado: tags=all,cart,pr,corpc,five_node
```

Comment:
what is a 'pr' tag?

Reply:
all tests in the functional vm stage are triggered by "pr,-hw" i.e. all tests with pr minus hw. so for cart tests to be triggered, it needs this tag